test/TestLintRule.py

```diff
@@ -18,23 +18,28 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-# pylint: disable=preferred-module  # FIXME: remove once migrated per GH-725
-import unittest
+import pytest
 
 from ansiblelint.file_utils import Lintable
 
-from .rules import EMatcherRule, UnsetVariableMatcherRule
+from .rules import EMatcherRule, RawTaskRule
 
 
-class TestRule(unittest.TestCase):
-    def test_rule_matching(self) -> None:
-        ematcher = EMatcherRule.EMatcherRule()
-        lintable = Lintable('examples/playbooks/ematcher-rule.yml', kind="playbook")
-        matches = ematcher.matchlines(lintable)
-        assert len(matches) == 3
+@pytest.fixture
+def lintable() -> Lintable:
+    """Return a playbook Lintable for use in this file's tests."""
+    return Lintable('examples/playbooks/ematcher-rule.yml', kind="playbook")
 
-    def test_rule_postmatching(self) -> None:
-        rule = UnsetVariableMatcherRule.UnsetVariableMatcherRule()
-        lintable = Lintable('examples/playbooks/bracketsmatchtest.yml', kind="playbook")
-        matches = rule.matchlines(lintable)
-        assert len(matches) == 2
```

Comment:
This used to be a test for `postmatch()` which doesn't exist any more. https://github.com/ansible-community/ansible-lint/commit/a4c6725aa4d38e7dca33b428f61cf1c885adc4f6#diff-d64893433e4ff6f626cc9b23e20b781c3ebb65b4df183c75dd536461f1076dc4R15

So, this `test_rule_postmatching` test duplicates `test_rule_matching`. So I deleted it.